cql-ast/src/main/scala/troy/cql/ast/Statements.scala

```diff
@@ -0,0 +1,59 @@
+package troy.cql.ast
+
+import troy.cql.ast.dml._
+import troy.cql.ast.ddl._
+
+sealed trait DataDefinition
+sealed trait DataManipulation
```

Comment:
Can you please change DataDefinition and DataManipulation to extend Cql3Statement?